atsign-foundation/at_tools
# Hive recovery

**- What I did**

Create a tool for recovering corrupted hive storage

**- How I did it**

- forked hive repo and commented the line of code preventing hive box from being opened.
- in the recovery tool, added forked hive repo as dependency
- recovery tool will attempt to open the corrupted box

**- How to verify it**
- take any corrupted storage from prod
- run the command
dart bin/main.dart < atsign > <dir_path_of_hive_box>
- you should see that open was opened succesfully and recovery result is true

**- Description for the changelog**
